packages/components/src/List/Toolbar/ColumnChooserButton/README.md

```diff
@@ -0,0 +1,274 @@
+Else, if children is not a function it will render normally.
+It consumes the [columnChooserContext](#columnchoosercontext).
+
+The component holds reference :
+* Row : [ColumnChooserRowRenderer](#columnchooserrowrenderer)
+
+| Props | Type | Info
+|---|---|---|
+| children | function | the body's  content of the popover, called with the columns as arguments.
+
+### ColumnChooserRowRenderer
+
+The row renderer only render its children.
+It holds reference to other components: , .
+By default, it consumes the [columnChooserContext](#columnchoosercontext).
+
+The component holds reference :
+* Visibility: [RowCheckbox](#rowcheckbox)
+* Label: [RowLabel](#rowlabel)
+
+| Props | Type | Info
+|---|---|---|
+| className | string | class passed to div wrapper row
+| children | React.element, [React.element] | the row's  content of the popover
+
+### RowCheckbox
+
+Add a checkbox to switch the visibility of a column.
+If the column is locked, a locked icon replaces the checkbox.
+	dataFeature,
+	describedby,
+	description,
+	id,
+	label,
+	locked = false,
+	onClick,
+	checked = false,
+
+| Props | Type | Info
+|---|---|---|
+| dataFeature | string | data-feature attribute of the input
+| describedby | string | the id of the markup holding the description
+| description | string | the description
+| id | string | id of the checkbox input
+| label | string | label used
+| locked | bool | indicates if the column is locked or not
+| onChange | func | trigger when the checkbox is clicked
+| checked | bool | indicates the visibility value, and if the checkbox is checked
+
+### RowLabel
+
+Add the label of the column to the row.
+
+| Props | Type | Info
+|---|---|---|
+| label | string | the label of the column
+
+## Hooks
+### ColumnChooserManagerHook
+
+This hook manages all the logic behavior in the [ColumnChooser](#columnchooser).
+It holds the state of the columns edited by the user.
+
+| Parameters | Type | Info
+|---|---|---|
+| initColumns | array | array of columns object from the list
+| nbLockedLeftItems | number | the number of columns that will be locked, left to right
+
+And it returns an object with hook state values and handlers to manipulate them.
+
+| Fields | Type | Info
+|---|---|---|
+| onChangeVisibility | function | change the visibility of the given column index|
+| onSelectAll | function | change the visibility of every columns
+| columnsChooser | [ColumnsChooserPropTypes](#columnschooserproptypes) | the hook columns
+| selectAll | bool | state of the select all button, checked or not.
+
+## Context
+
+### columnChooserContext
+
+The column chooser uses a context to pass some props to its children and avoid props drilling.
+By default the context is initialized in [ColumnChooser](#columnchooser).
+You can import directly the provider and the consumer from columnChooser.context
+```
+import { columnChooserContext, ColumnChooserProvider } 
+															from '.../columnChooserContext'
```

Comment:
indentation issue, and it doesn't appear in the page
<img width="527" alt="Capture d’écran 2019-08-06 à 11 21 07" src="https://user-images.githubusercontent.com/10761073/62527919-5380a880-b83c-11e9-96c1-3851ab6229d6.png">